studio/components/interfaces/BillingV2/Invoices.tsx

```diff
@@ -103,7 +68,7 @@ const Invoices = ({ projectRef }: InvoicesProps) => {
               <Table.tr>
                 <Table.td colSpan={6} className="p-3 py-12 text-center">
                   <p className="text-scale-1000">
-                    {loading ? 'Checking for invoices' : 'No invoices for this project yet'}
+                    {isLoading ? 'Checking for invoices' : 'No invoices for this project yet'}
```

Comment:
`isLoading` and `isSuccess` can't both be true, right?

Reply:
nope - the states from RQ are all mutually exclusive (isLoading, isSuccess, isError for the one's that we commonly use, and all the other states too)

Reply:
ohh sorry Alaister just pointed out that isLoading will never be true here since its within the isSuccess conditional - will update! 🙏